dev/integration_tests/external_ui/test_driver/main_test.dart

```diff
@@ -32,37 +32,37 @@ Future<void> main() async {
       await driver.waitFor(fab);
 
       final String calibrationResult = await driver.getText(summary);
-      final Match matchCalibration = calibrationRegExp.matchAsPrefix(calibrationResult);
+      final Match? matchCalibration = calibrationRegExp.matchAsPrefix(calibrationResult);
       expect(matchCalibration, isNotNull);
-      final double flutterFrameRate = double.parse(matchCalibration.group(1));
+      final double flutterFrameRate = double.parse(matchCalibration?.group(1) ?? '0');
 
       // Texture frame stats at 0.5x Flutter frame rate
       await driver.tap(fab);
       await Future<void>.delayed(samplingTime);
       await driver.tap(fab);
 
       final String statsSlow = await driver.getText(summary);
-      final Match matchSlow = statsRegExp.matchAsPrefix(statsSlow);
+      final Match? matchSlow = statsRegExp.matchAsPrefix(statsSlow);
       expect(matchSlow, isNotNull);
-      expect(double.parse(matchSlow.group(1)), closeTo(flutterFrameRate / 2.0, 5.0));
-      expect(double.parse(matchSlow.group(2)), closeTo(flutterFrameRate / 2.0, 5.0));
-      expect(int.parse(matchSlow.group(3)), 1);
+      expect(double.parse(matchSlow?.group(1) ?? '0'), closeTo(flutterFrameRate / 2.0, 5.0));
+      expect(double.parse(matchSlow?.group(2) ?? '0'), closeTo(flutterFrameRate / 2.0, 5.0));
```

Comment:
Maybe make `Match matchSlow` nonnull with a `!` and another `!` on the group instead of the `?? '0'`.

Reply:
We can't make `Match matchSlow` because `matchAsPrefix` returns `Match?` @jmagman

Reply:
Yup, I meant since this is a test:
```dart
final Match matchSlow = statsRegExp.matchAsPrefix(statsSlow)!;
expect(double.parse(matchSlow.group(1)!), closeTo(flutterFrameRate / 2.0, 5.0));
expect(double.parse(matchSlow.group(2)!), closeTo(flutterFrameRate / 2.0, 5.0));
```
What you have is fine though, up to you.

Reply:
Nice idea
let me do this, please
thanks